### DISCUSSION TITLE
zipfile.BadZipFIle: File is not a zip file

### DISCUSSION THREAD

**Discussion by mukulmeena** [Atri-Labs/atrilabs-engine#608] [Help: Installation & Start] [answered]
After running Atri start command, it downloads the atri engine and when it starts extracting it shows an error.

![atri app json - my_app - Visual Studio Code 22-12-2022 14_59_08](https://user-images.githubusercontent.com/84377586/209103661-8a81bd26-9a17-4c3a-9b83-966642e45589.png)

**Comment by ramchaudhary2000** [comment]
Same issue here

**Reply by mukulmeena** [reply]
@ramchaudhary2000 Did you find the fix?

**Comment by darshitac11** [comment, marked as answer]
Check the solution here: https://github.com/pypa/pipenv/issues/581
